packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -149,29 +155,123 @@ const useStyles = makeStyles<BackstageTheme>(
         justifyContent: 'center',
       },
       submenuArrow: {
+        display: 'flex',
+      },
+      expandButton: {
+        background: 'none',
+        border: 'none',
+        color: theme.palette.navigation.color,
+        width: '100%',
+        cursor: 'pointer',
+        position: 'relative',
+        height: 48,
+      },
+      arrows: {
         position: 'absolute',
-        right: 0,
+        right: 10,
       },
       selected: {
         '&$root': {
           borderLeft: `solid ${selectedIndicatorWidth}px ${theme.palette.navigation.indicator}`,
           color: theme.palette.navigation.selectedColor,
         },
-        '&$closed': {
+        buttonItem: {
+          background: 'none',
+          border: 'none',
+          width: 'auto',
+          margin: 0,
+          padding: 0,
+          textAlign: 'inherit',
+          font: 'inherit',
+        },
+        closed: {
           width: drawerWidthClosed,
+          justifyContent: 'center',
+        },
+        open: {
+          width: drawerWidthOpen,
+        },
+        label: {
+          // XXX (@koroeskohr): I can't seem to achieve the desired font-weight from the designs
+          fontWeight: 'bold',
+          whiteSpace: 'nowrap',
+          lineHeight: 'auto',
+          flex: '3 1 auto',
+          width: '110px',
+          overflow: 'hidden',
+          'text-overflow': 'ellipsis',
         },
-        '& $closedItemIcon': {
-          paddingRight: selectedIndicatorWidth,
+        iconContainer: {
+          boxSizing: 'border-box',
+          height: '100%',
+          width: iconContainerWidth,
+          marginRight: -theme.spacing(2),
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
         },
-        '& $iconContainer': {
-          marginLeft: -selectedIndicatorWidth,
+        searchRoot: {
+          marginBottom: 12,
+        },
+        searchField: {
+          color: '#b5b5b5',
+          fontWeight: 'bold',
+          fontSize: theme.typography.fontSize,
+        },
+        searchFieldHTMLInput: {
+          padding: `${theme.spacing(2)} 0 ${theme.spacing(2)}`,
```

Comment:
the spacing helper is pretty nifty and helpful, it takes multiple arguments and deals with making the output correct. This one can be replaced with `padding: theme.spacing(2, 0, 2)`. You can search for `{theme.spacing(` and find a couple more like this